app/assets/javascripts/modules/track-briefing.js

```diff
@@ -0,0 +1,34 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function TrackBriefing () {}
+
+  TrackBriefing.prototype.start = function($element) {
+    var scope = $element[0]
+
+    this.trackLinks(scope)
+  }
+
+  TrackBriefing.prototype.trackLinks = function(scope) {
+    var links = scope.getElementsByTagName('a')
```

Comment:
Do we wanna make this for all links? Or only external links to gov.uk? I assume we collect user journey anyways when they stay on gov.uk domain.

Reply:
We already collect external - I will ask the PA if we want both

Reply:
PA said all links